src/api/backend_connectors/database_connector.py

```diff
@@ -201,15 +227,26 @@ def add_message(text: str, role: str, cid: str) -> None:
             raise DatabaseException("Something went wrong.", UNKNOWN_ISSUE)
 
 
-def end_conversation(uid: str, cid: str) -> None:
+def end_conversation(cid: str) -> None:
     """Delete conversation.
 
     Arguments:
-        uid (str): User id.
         cid (str): Conversation id.
 
+    Raises:
+        DatabaseException
+
     """
-    pass
+    db_config = _load_config()
```

Comment:
Should we catch the potential databaseexception from _load_config? Otherwize the API will crash if the exception is raised?

Reply:
Jao, kommer få gå igenom alla fel som kan uppstå.